sdk/devcenter/azure-developer-devcenter/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+directory: specification/devcenter/DevCenter
+commit: 74e93fc9fb00380250110fba78dadfebeb93cbf1
+repo: Azure/azure-rest-api-specs
```

Comment:
It is not on main https://github.com/Azure/azure-rest-api-specs/tree/main/specification/devcenter/DevCenter

If the code is generated from typespec, why I didn't see any new model class in PR? This is strange...

Reply:
OK. I see you turned off all convenienceAPI https://github.com/Azure/azure-rest-api-specs/blob/74e93fc9fb00380250110fba78dadfebeb93cbf1/specification/devcenter/DevCenter/client.tsp

Reply:
Yes, we decided to not generate with models to have the least diff possible and speed up the process since it will introduce less changes. The plan is generate with models for the stable SDK

Reply:
The commit ID should be on spec main branch.

If the commit is not there, you kind of releasing an SDK from your own branch or your own PR. It is almost impossible to track what's the source.

Reply:
Oh yes, I have created an issue https://github.com/Azure/azure-sdk-for-net/issues/39511  in the .net repo to address that. Once we have it in main I should have a new PR for each languange updating the commit hash in the tsp-location.yaml